Jamulus.pro

```diff
@@ -738,6 +739,7 @@ DISTFILES += ChangeLog \
     src/res/translation/translation_it_IT.qm \
     src/res/translation/translation_sv_SE.qm \
     src/res/translation/translation_sk_SK.qm \
+    src/res/translation/translation_zh_CN.qm \
```

Comment:
I'm not 100% sure, but as far as I know, the .qm files should not be committed via this PR? They'd be generated once we're doing a release.

Reply:
Actually part of the reason I commit the qm file is because the binaries build by CI could get the translation right away, and I can then redistribute the CI build to my friends who cannot speak English but would like to use the app right now.

I don't plan to submit translation updates very frequently once this patch get merged, I will probably do translation only before there will be a release so hopefully my work won't break your release cycle.

If it's not okay for this patch, let me know and I'll remove the qm file from this PR, thanks!

Reply:
I‘m not that involved in the translation process, so I‘m not sure how to deal with it. Probably it’s fine to have them there.

Reply:
There is no problem committing the .qm file. Having the updated .qm file does help when building under older versions of Qt, as they don't automatically compile them when building.

There was some motivation for removing the .qm files from git and only compiling them when building, but there are complications in doing that which were never solved satisfactorily (#1303)